infra/scoped/ecs_services.tf

```diff
@@ -0,0 +1,44 @@
+locals {
+  requests_container_port = 9001
+}
+
+module "requests" {
+  source = "../modules/ecs_service"
+
+  service_name            = "requests-${terraform.workspace}"
+  deployment_service_name = "requests"
+
+  container_image        = "${local.requests_repository}:env.${terraform.workspace}"
+  container_port         = local.requests_container_port
+  deployment_service_env = terraform.workspace
+  desired_task_count     = 1
```

Comment:
Why only 1?

Reply:
true, I'll make this the same as how we do tasks and AZs in the catalogue API